roles/mail/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+dependencies:
+  - { role: kamaln7.swapfile, swapfile_use_dd: True, swapfile_size: 2048, swapfile_swappiness: 10 }
```

Comment:
What is the reason for this swapfile configuration?

Reply:
The spam filters are consuming a lot of memory. Since the main testing host was t2.micro with 1 GiB RAM, there were cases where amavis has run out of memory and system terminated it resulting in non-working mail server.